internal/controller/datadogagentinternal/controller_reconcile_agent.go

```diff
@@ -66,7 +66,7 @@ func (r *Reconciler) reconcileV2Agent(logger logr.Logger, requiredComponents fea
 
 		// Apply features changes on the Deployment.Spec.Template
 		for _, feat := range features {
-			if errFeat := feat.ManageNodeAgent(podManagers); errFeat != nil {
+			if errFeat := feat.ManageNodeAgent(podManagers, ""); errFeat != nil {
```

Comment:
We use the no provider `""` as the ManageNodeAgent signature from dda includes the provider string for introspection, that is not supported with DDAI yet.